### ISSUE TITLE
SyntaxError: Unexpected token /

### ISSUE THREAD

**Issue by tomleo** [erming/shout#408]
````
root@cloudleo:~# shout

Shout is now running on http://0.0.0.0:9000/
Press ctrl-c to stop

[SyntaxError: Unexpected token /]

root@cloudleo:~# node --version
v0.10.38
```

Is this a bug, or a configuration issue on my end?

**Comment by erming** [comment]
Never seen this error before. I'm running v0.10.25 myself. I'll get the latest version and see if that's the problem.

You get this every time? And when? When you connect to http://0.0.0.0:9000/ from your browser or instantly when starting the Shout server?

**Comment by tomleo** [comment]
I get this every time when I start shout from the command line as shown above.

```
root@cloudleo:~# cat /etc/*-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=14.04
DISTRIB_CODENAME=trusty
DISTRIB_DESCRIPTION="Ubuntu 14.04.2 LTS"
NAME="Ubuntu"
VERSION="14.04.2 LTS, Trusty Tahr"
ID=ubuntu
ID_LIKE=debian
PRETTY_NAME="Ubuntu 14.04.2 LTS"
VERSION_ID="14.04"
```

**Comment by erming** [comment]
I could not reproduce this error. But because the error message seems to be an exception without a stack trace, I went looking for any stray `console.log` inside in the code.

Found one in the client manager, which loads user `.json`-files. I guess that's where your error messages is coming from.

Seems there's something wrong in your json files. Try to fix it manually (go to `~/.shout/users/` and edit your user files) or just remove the faulty user and re-create it.
